tests/plans/das-benchmarks/001-lights-dasing-latest-from-bridge.go

```diff
@@ -0,0 +1,29 @@
+package dasbenchs
+
+import (
+	dasbenchmarks "github.com/celestiaorg/test-infra/tests/helpers/das-benchmarks"
+	"github.com/testground/sdk-go/run"
+	"github.com/testground/sdk-go/runtime"
+)
+
+// Test-Case #001 - X Light Nodes have finished DASing (from one full/bridge node) before block time
+// Description is in docs/test-plans/002-das-benchmark/test-cases
+func LightsDasingLatest(runenv *runtime.RunEnv, initCtx *run.InitContext) (err error) {
+	switch runenv.StringParam("role") {
+	case "validator":
+		err = dasbenchmarks.RunValidator(runenv, initCtx)
+	case "bridge":
+		err = dasbenchmarks.RunBridgeNode(runenv, initCtx)
+	case "light":
+		err = dasbenchmarks.RunLightNode(runenv, initCtx)
+	}
+
+	if err != nil {
+		runenv.RecordFailure(err)
```

Comment:
if we caught the err and didn't signaled of finished state -> the test will be killed because of the daemon's runtime worker timeout. Please remember to wrap errors with a signal state